### PR TITLE
Standardize layer number message

### LINKED ISSUE #88: layer number data type
It appears to be uint32 in some places, uint64 in others. Let's pick one and standardize.

https://github.com/spacemeshos/api/blob/a91231e795693339a4bda4ea1b70df4b25d8919d/proto/spacemesh/v1/mesh_types.proto#L89-L90

https://github.com/spacemeshos/api/blob/a91231e795693339a4bda4ea1b70df4b25d8919d/proto/spacemesh/v1/types.proto#L115

CC @avive

### PR DESCRIPTION
**Pull request by lrettig**
Closes #88

### CODE REVIEW

**Comment by avive** [comment]
I see you chose uint64 for LayerNumber. Why not uint32?

File: proto/spacemesh/v1/types.proto [hunk 1]
```diff
@@ -125,6 +125,10 @@ message Layer {
     bytes root_state_hash = 6; // when available - the root state hash of global state in this layer
 }
 
+message LayerNumber {
+    uint64 number = 1;
```
Comment (avive) [inline proto/spacemesh/v1/types.proto:129]: Why uint64 and not uint32 for this?

**Comment by lrettig** [comment]
> See my question about uint32 vs unit64 

See https://github.com/spacemeshos/api/issues/88#issuecomment-655632758

**Comment by lrettig** [comment]
Needs https://github.com/spacemeshos/go-spacemesh/issues/2082

**Pull request merged**